### DISCUSSION TITLE
zh/guide/drivers/ftp

### DISCUSSION THREAD

**Discussion by giscus** [AlistGo/docs#50] [Announcements]
# zh/guide/drivers/ftp

AList v3的文档

https://alist.nn.ci/zh/guide/drivers/ftp.html

**Comment by doullse** [comment]
Failed init storage: dial tcp ***********57:21: connect: connection refused

**Reply by doullse** [reply]
这是windows开启ftp后，用alist连接提示的错误，经过其他设备测试是可以正常连接到windows的ftp的。

**Reply by zhaowei05** [reply]
直接写 ip+:端口 这种 127.0.0.1:21，不用前面的ftp://

**Comment by hitptep** [comment]
ftp的挂载路径怎么写哇

**Reply by iamqiz** [reply]
随便取,比如FTP手机

**Comment by hitptep** [comment]
Failed create storage in database: UNIQUE constraint failed: x_storages.mount_path

**Comment by xiureal** [comment]
请问那个FTP文字编码没有gb2312，挂在FTP，中文显示乱码

**Reply by Richman881** [reply]
同问，编码不对，中文显示乱码

**Comment by simosun** [comment]
根文件夹路径    
配置了，没有变？填个错误的，也报错？

**Comment by gufengxingyu** [comment]
Non-anonymous sessions must use encryption

**Comment by flyweb291** [comment]
FTP配置保存成功，一直转圈圈

**Comment by flyweb291** [comment]
端口填的21，但是报错：failed get objs: failed to list objs: dial tcp ************:39993: connect: connection timed out

**Comment by 573648656** [comment]
成功了，但是在目录打开提示failed get objs: failed to list objs: 550 Can't check for file existence

**Comment by ayacnc** [comment]
使用padavan内建的ftp服务器挂载后，只能打开名为AiDisk_a1的目录，进入后无内容显示。

**Reply by ayacnc** [reply]
电脑端使用FlashFXP，手机端使用Kodi及Mixplorer均能正常通过ftp连接，唯独alist挂载后无法显示目录

**Comment by xkatld** [comment]
这个模式会走主站机的流量吗？还是说走节点流量

**Comment by shinobiz** [comment]
不能更改字符编码？2台ftp服务器均成功挂载至Alist，但是客户端通过Alist访问这两台ftp服务器播放视频加载很久，拖动也是，有的时候根本加载不了。走smb协议没问题。客户端直接ftp访问2台服务器也没问题。怀疑还是Alist没有提供ftp字符编码设置导致。
